commands/paste-emoji/command.toml

```diff
@@ -0,0 +1,16 @@
+name = "Paste Emoji"
+description = "Lets you search through emojis and paste it into your current applciation"
+endpointURL = "https://slapcommands.vercel.app/api/utils/emoji-paster"
+sourceCodeURL = "https://github.com/AnishDe12020/slapcommands/blob/main/src/shared/utils/emoji-paster.js"
+language = "javascript"
+categories = ["Utilities", "Productivity"]
```

Comment:
While for some people emojis can belong to "Productivity", I think it will probably be best to leave only "Utilities"  for now.